mewarpx/mewarpx/diags_store/field_diagnostic.py

```diff
@@ -150,12 +150,12 @@ def fields_diag(self):
                 data = np.abs(
                     mwxrun.get_gathered_rho_grid(
                         species_name=species.name, include_ghosts=False
-                    ) / mwxconstants.e * 1e-6
```

Comment:
The `species.sq` will need to go inside the `np.abs` (or maybe you can just remove the absolute value call).